metrics.md

```diff
@@ -295,11 +295,52 @@ Uncertainty is the variance of the event indicator $$ o(t) $$. Low values of UNC
 {: .anchor }
 The BSS is based on the BS and measures the performance of a probability forecast relative to a reference forecast:
 
-$$ BSS = 1 - \frac{\text{BS}_f}{\text{BS}_{\text{ref}}} $$
+$$ \text{BSS} = 1 - \frac{\text{BS}_f}{\text{BS}_{\text{ref}}} $$
 
 where $$ \text{BS}_f $$ is the BS of the forecast of interest, and $$ \text{BS}_{\text{ref}} $$ is the BS of the reference forecast. BSS greater than zero indicates the forecast performed better than the reference and vice versa for BSS less than zero, while BSS equal to zero indicates the forecast is no better (or worse) than the reference.
 
 
+### Quantile Score (QS) {#qs}
+{: .anchor }
+QS measures the accuracy of quantile forecasts, in which the forecast predicts the variable value corresponding to a constant probability ([Koenker78](#ref-koenker78), [Bouallegue15](#ref-bouallegue15)). QS is similar to BS, but measures accuracy in terms of the variable value (e.g. MW) and is defined as:
+
+$$ \text{QS} = \frac{1}{n} \sum_{i=1}^n (\text{fx}_i - \text{obs}_i) \cdot (p - \mathbf{1}\{ \text{obs}_i > \text{fx}_i \}) $$
+
+where $$ \mathbf{1}\{ \text{obs} > \text{fx} \} $$ is an indicator function:
+
+$$ \mathbf{1}\{ \text{obs} > \text{fx} \} = \begin{cases}
+    \displaystyle 1 & \text{obs} > \text{fx}  \\
+    \displaystyle 0 & \text{obs} \leq \text{fx}
+\end{cases} $$
+
+If $$ \text{obs} > \text{fx} $$, then QS is non-negative:
+
+$$\begin{align}
+    (\text{fx} - \text{obs}) &< 0 \\
+    (p - \mathbf{1}\{\text{obs} > \text{fx}\}) &= (p - 1) \leq 0 \\
+    (\text{fx} - \text{obs}) \cdot (p - 1) &\geq 0
+\end{align} $$
+
+If instead $$ \text{obs} < \text{fx} $$, then QS is also non-negative:
+
+$$\begin{align}
+    (\text{fx} - \text{obs}) &> 0 \\
+    (p - \mathbf{1}\{\text{obs} > \text{fx}\}) &= (p - 0) \geq 0 \\
+    (\text{fx} - \text{obs}) \cdot p &\geq 0 \\
+\end{align} $$
+
+Based on the above definitions, smaller QS values indicate more accurate forecasts.
```

Comment:
```suggestion
Smaller QS values indicate more accurate forecasts.

QS is always greater than or equal to 0. If $$ \text{obs} > \text{fx} $$, then QS is non-negative:

$$\begin{align}
    (\text{fx} - \text{obs}) &< 0 \\
    (p - \mathbf{1}\{\text{obs} > \text{fx}\}) &= (p - 1) \leq 0 \\
    (\text{fx} - \text{obs}) \cdot (p - 1) &\geq 0
\end{align} $$

If instead $$ \text{obs} < \text{fx} $$, then QS is also non-negative:

$$\begin{align}
    (\text{fx} - \text{obs}) &> 0 \\
    (p - \mathbf{1}\{\text{obs} > \text{fx}\}) &= (p - 0) \geq 0 \\
    (\text{fx} - \text{obs}) \cdot p &\geq 0 \\
\end{align} $$
```

Reply:
To me, the most important thing is that smaller QS is better. The second most important thing is that it's >=0. The proof is the least interesting. So my suggestion reorders the text to match that. Feel free to disagree.

Reply:
That makes sense to me.